packages/filter/src/components/moreFilterSelect/MoreFilterSelect.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import Button from '@xcritical/button';
+import Select from '@xcritical/select';
+import { OptionTypeBase } from 'react-select';
+
+import {
+  Dropdown,
+} from '../Dropdown';
+
+import {
+  ChevronDown,
+  ChevronUp,
+  DropdownIndicator,
+  Plus,
+} from '../icons';
+import {
+  IMoreButtonWithFilterSelectorProps,
+  IStateRecivedFilter,
+} from '../../interfaces';
+import {
+  convertFiltersToOptions,
+  convertSelectedFiltersToOptions,
+} from '../../utils';
+
+
+const selectStyles = {
+  control: (provided: any) => ({ ...provided, minWidth: 240, margin: 8 }),
+  menu: () => ({ boxShadow: 'inset 0 1px 0 rgba(0, 0, 0, 0.1)' }),
+};
+
+export const MoreFilterSelect: React.FC<IMoreButtonWithFilterSelectorProps> = ({
+  filters,
+  selectedFilters,
+  onChange,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+
+  const filterItems = useMemo(
+    () => convertFiltersToOptions(filters),
+    [filters],
+  );
+
+  const selectedValueItems = useMemo(
```

Comment:
selectedValueItems -> selectedValueOptions

Reply:
It doesn't matter